packages/menu-bar/src/vaadin-menu-bar-interactions-mixin.js

```diff
@@ -247,6 +295,12 @@ export const InteractionsMixin = (superClass) =>
         } else if (isOpened) {
           this._close();
         }
+
+        if (button === this._overflow) {
+          this._hideTooltip();
+        } else {
+          this._showTooltip(button);
```

Comment:
We need to check out if `openOnHover` is true, as in that case the tooltip shouldn't be shown.

Reply:
Done